hazelcast/src/main/java/com/hazelcast/cache/impl/journal/RingbufferCacheEventJournalImpl.java

```diff
@@ -180,9 +191,16 @@ public RingbufferConfig toRingbufferConfig(EventJournalConfig config) {
                 .setTimeToLiveSeconds(config.getTimeToLiveSeconds());
     }
 
-    private void addToEventRingbuffer(ObjectNamespace namespace, int partitionId, CacheEventType eventType,
-                                      Data key, Object oldValue, Object newValue) {
-        final RingbufferContainer<InternalEventJournalCacheEvent> eventContainer = getRingbufferOrNull(namespace, partitionId);
+    private void addToEventRingbuffer(EventJournalConfig journalConfig, ObjectNamespace namespace, int partitionId,
```

Comment:
minor: do you see any other path that can result with exception in this method?

Reply:
I went through the rest of the code but I don't see any obvious ways of failing